src/test/invalid_jump.c

```diff
@@ -7,9 +7,13 @@ static void sighandler(int sig) {
   _exit(0);
 }
 
-char invalid_jump_here[] = { 0x00, 0x00, 0x00, 0x00, 0x00 };
-
 int main(void) {
+  char* invalid_jump_here;
+  size_t page_size = sysconf(_SC_PAGESIZE);
+
+  invalid_jump_here = mmap(NULL, page_size, PROT_NONE, MAP_ANONYMOUS | MAP_PRIVATE, -1, 0);
```

Comment:
Add a cast from void*. I'm not sure how this compiles.

Reply:
Cast added and pushed.